src/chocolatey.resources/helpers/functions/Get-ChocolateyWebFile.ps1

```diff
@@ -261,7 +267,7 @@ param(
   if ($url.StartsWith('http:')) {
     try {
       $httpsUrl = $url.Replace("http://", "https://")
-      Get-WebHeaders -Url $httpsUrl -ErrorAction "Stop" | Out-Null
+      Get-WebHeaders -Url $httpsUrl -ErrorAction "Stop" -Credential $credential | Out-Null
```

Comment:
I think so (though I'm no powershell expert). Get-WebHeaders has a check for a $null cred (around line 71). Is that what you mean?